news.d/feature/989.core.rst

```diff
@@ -0,0 +1,6 @@
+A new ``SET_CONFIG`` command can be used to change the configuration with a stroke, e.g.::
+
+        "O*EP": "{PLOVER:SET_CONFIG:translation_frame_opacity,100}",
+        "TR*PB": "{PLOVER:SET_CONFIG:translation_frame_opacity,0}",
```

Comment:
This needs to be updated to show the correct format

```suggestion
        "O*EP": "{PLOVER:SET_CONFIG:'translation_frame_opacity':100}",
        "TR*PB": "{PLOVER:SET_CONFIG:'translation_frame_opacity':0}",
```

Reply:
Good catch, amended.